### PR TITLE
[TE] rootcause improvements and bug fixes

### PR DESCRIPTION
**Pull request by justYves**
### What's new: 
- Fixed rootcause anomaly tooltip z-index 
- Prettified Anomaly change Format and improve responsive wrapping logic
- Added id# to anomalies
- Apply proper class to events table

### Test: 
Visual sanity check

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/components/update-table/component.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ import modelsTable from 'ember-models-table/components/models-table';
  * the table's selected items should reflect this
  */
 export default modelsTable.extend({
+
+  classNames: ['events-table'],
```
Comment (ttbach) [inline thirdeye/thirdeye-frontend/app/pods/components/update-table/component.js]: the `metrics-table` also uses `update-table`. Can you double check that there are no side effects here?

File: thirdeye/thirdeye-frontend/app/helpers/utils.js [hunk 2]
```diff
@@ -265,6 +265,24 @@ export function toMetricLabel(urn, entities) {
...
+ * Returns a human-readable label for an event urn
+ *
+ * @param {string} urn event urn
+ * @param {Object} entities entities cache
+ * @returns {string} human-readable event label
+ */
+export function toEventLabel(urn, entities) {
+  let label = entities[urn].label;
+
+  if (urn.includes('anomaly')) {
```
Comment (apucher) [inline thirdeye/thirdeye-frontend/app/helpers/utils.js]: I should address this in the backend in the long run

File: thirdeye/thirdeye-frontend/app/helpers/format-number.js [hunk 3]
```diff
@@ -1,12 +1,17 @@
...
  * Displays human readable number
  * @param {number} num A number
  * @return {string} human readable number
  */
-export function formatNumber([num=0]/*, hash*/) {
+export function formatNumber([num=0], {formatter, format}) {
+  if (formatter && format) {
+    return d3.format(format)(num);
+  }
+
```
Comment (ttbach) [inline thirdeye/thirdeye-frontend/app/helpers/format-number.js]: Can we include an `@example` here?

File: thirdeye/thirdeye-frontend/app/pods/components/update-table/component.js (hunk 1 above)
Comment (justYves) [reply thirdeye/thirdeye-frontend/app/pods/components/update-table/component.js]: sure thing.

File: thirdeye/thirdeye-frontend/app/helpers/format-number.js (hunk 3 above)
Comment (justYves) [reply thirdeye/thirdeye-frontend/app/helpers/format-number.js]: done!

**Pull request merged**